draft-packages/modal/KaizenDraft/Modal/Presets/RoadblockModal.scss

```
@@ -1,40 +1,46 @@
-@import "~@kaizen/design-tokens/sass/color";
-@import "~@kaizen/design-tokens/sass/border";
-@import "~@kaizen/design-tokens/sass/spacing";
+@import "~@kaizen/design-tokens/sass/color-vars";
+@import "~@kaizen/design-tokens/sass/border-vars";
+@import "~@kaizen/design-tokens/sass/spacing-vars";
+@import "~@kaizen/design-tokens/sass/variable-identifiers";
 @import "~@kaizen/deprecated-component-library-helpers/styles/color";
 @import "~@kaizen/component-library/styles/layout";
 @import "~@kaizen/component-library/styles/responsive";
 
+$dt-roadblock-header-background: var(
```

It might be worth including a comment with the "zen" keyword so these are easier to clean up afterward.

```suggestion
// Use Peach 100 for Heart theme, and Peach 500 for Zen (default fallback)
$dt-roadblock-header-background: var(
```